### ISSUE TITLE
Model turns green  and black when I make the first click on the model in retopoflow, and I can't see what I am doing

### ISSUE THREAD

**Issue by Zoglow** [CGCookie/retopoflow#638]

**Comment by vxlcoder** [comment]
Thanks for the report @Zoglow.  We will need more information before we can help in debugging this issue.
Can you post a screenshot showing what it looks like before and after you click.
Which version of Blender are you using?  Which version of RetopoFlow are you using?

**Comment by jlampel** [comment]
This sounds like a duplicate of the v2 MacOS visualization error. Closing out due to inactivity.
